### DISCUSSION TITLE
Changing "accessed" word in the bibliography

### DISCUSSION THREAD

**Discussion by Cano707** [typst/typst#1571] [Q&A] [answered]
Hi, 

I would like to ask, if it is possible to change the "accessed" word, which indicates when a certain web source has been visited the last time.

For example, the following BibTeX entry:
```
@online{owllink,
  title = {{SOME_TITLE}},
  url = {SOME_URL},
  urldate = {2023-06-25}
}
```
results in 
```
“SOME_TITLE” SOME_URL (accessed: Jun. 25, 2023).
```

I would like to change the word "accessed" to something else.

I appreciate your time.

**Comment by Moneyless7447** [comment]
I would like to know the answer, too, since I am facing the same issue

**Comment by laurmaedje** [comment, marked as answer]
There is no clean way. However, you can write an `#show "accessed": "something else"` show rule before the bibliography. It will affect every occurance of the word "accessed" though (also in titles or wherever).
